settings/test.py

```diff
@@ -5,6 +5,7 @@
 DATABASES = {
     "default": {
         "ENGINE": "django.db.backends.postgresql_psycopg2",
+        "ENGINE": "django_prometheus.db.backends.postgresql",
```

Comment:
Why are we adding multiple `ENGINE` keys in the `DATABASES` config? This will break EvalAI DB access